app/src/main/java/com/yterletskyi/happyfriend/features/settings/domain/SettingEnum.kt

```diff
@@ -3,5 +3,5 @@ package com.yterletskyi.happyfriend.features.settings.domain
 enum class SettingEnum {
     MY_WISHLIST,
     APP_VERSION,
-    GENERAL_LIST,
+    GENERAL_IDEAS,
```

Comment:
sometimes in code it's GENERAL_IDEA`S` and sometimes `general idea`. Let's stick to one name everywhere -- _general ideas_ (plural)